includes/payments/actions.php

```diff
@@ -171,8 +171,7 @@ function edd_undo_purchase_on_refund( $payment_id, $new_status, $old_status ) {
 
 	if( $customer_id ) {
 
-		$customer = new EDD_Customer( $customer_id );
-		$customer->decrement_stats( $customer_id, $amount );
+		EDD()->customers->decrement_stats( $customer_id, $amount );
```

Comment:
Please do not use ```EDD->customers``` to do this. Please the proper method to remove payments from a customer, either by individually updating the purchase count and purchase value...or the ```remove_payment``` method:

```$customer->remove_payment( $payment_id );